### ISSUE TITLE
Rat AI doesnt dig moat

### ISSUE THREAD

**Issue by SirMars1992** [Sh0wdown/UnofficialCrusaderPatch#598]
Has anyone else the same problem?
The Rat AI isnt able to dig his own moat... is there a possibility to fix that or did I do something wrong?

**Comment by Truetobi91** [comment]
Yes it is possible. You need to modify an own aic file that gives the rat troops to do so. Look for aic tutorials and the information you find here in the wiki.

**Comment by J-T-de** [comment]
The aic parameter are called "DefDiggingUnit" and "DefDiggingUnitMax".

**Comment by Krarilotus** [comment]
usually the rat will use its starting speerman to dig moat if i remember correctly. But it will probably not replace those in vanilla?

**Comment by Heroesflorian** [comment]
@Krarilotus vanilla Rat doesn't dig moat... not even with starting troops.
